Sources/Operation.swift

```diff
@@ -36,11 +49,17 @@ struct OperationBuilder: Builder {
     let responses: [Int : Reference<ResponseBuilder>]
     let defaultResponse: Reference<ResponseBuilder>?
     let deprecated: Bool
+    let operationId: String?
```

Comment:
operationIdentifier
In fact, the 'operation' is inferred, maybe just call this and the one above 'identifier'